doc/source/examples/sharp-immersed-boundary/sedimentation-1-cuboid/sedimentation-1-cuboid.rst

```diff
@@ -0,0 +1,272 @@
+==============================================================================
+Sedimentation of One Cuboid Particle
+==============================================================================
+
+This example aims to simulate the S18 experiment of Wang `et al.` [#Wang2024]_. This experience measures the velocity of the sedimentation of a :math:`2\,\text{cm}` cuboid particle in a container filled with a glycerin-water solution.
+
+.. note::
+    This example is similar to the sedimentation of one particle example (:doc:`../sedimentation-1-particle/sedimentation-1-particle`) but with a cuboid particle instead of a sphere. As such, the physical setup changes, but the parameters relating to the numerical resolution remain very similar.
+
+.. warning:: 
+    * This case is a computationally expensive example. It can take several hours to run on a desktop computer.
+
+----------------------------------
+Features
+----------------------------------
+
+- Solvers: ``lethe-fluid-sharp`` (with Q1-Q1)
+- Transient problem
+- Displays the capability of the resolved CFD-DEM solver for the flow around one non-spherical particle
+
+
+---------------------------
+Files Used in This Example
+---------------------------
+
+- Parameter file: ``/examples/sharp-immersed-boundary/sedimentation-1-cuboid/sedimentation-1-cuboid.prm``
+
+
+-----------------------
+Description of the Case
+-----------------------
+
+The S18 experiment consists of the release of a cuboid particle made of stainless steel (:math:`\rho_p=0.00759 \frac{\text{kg}}{\text{cm}^{3}}`)  with sides of 2.0 cm dropped into a water column of dimensions :math:`40 \times 80 \times 40\,\text{cm}`. In their experiment, the authors use a glycerin-water solution with varying concentrations to get different settling regimes. The relevant properties to the S18 case, as reported by the authors, are the fluid kinematic viscosity :math:`\nu_f=1.6827 \frac{\text{kg}}{\text{s cm}}` and the fluid density :math:`\rho_f=0.00124 \frac{\text{kg}}{\text{cm}^{3}}`. The gravity constant is :math:`g= -981 \frac{\text{cm}}{\text{s}^{2}}`. The particle accelerates due to gravity until it hits the bottom of the container, at which point we stop the simulation.
+
```

Comment:
You can maybe add the terminal Reynolds number for the particle sedimentation to specify the flow regime.